.gitignore

```diff
@@ -92,9 +92,6 @@ node_modules/
 # Yarn Integrity file
 .yarn-integrity
 
-# Yarn Lock file
-yarn.lock
```

Comment:
Above in the android build you told it to not use the lock file. Is that a mistake?

Reply:
no, we have a conflict to resolve on that front. there's a mix of lock files and the behavior of a frozen lockfile results in everything failling. it will use the lockfile and we should move to that. documenting for posterity.